lib/mixins/_readable.js

```diff
@@ -153,6 +184,58 @@ _Readable.prototype.toFile = function(path, options) {
 	return this.pipe(fs.createWriteStream(path, options));
 };
 
+/**
+ * Splits the incoming data stream into discrete objects
+ *
+ * @method split
+ * @param {String|RegEx} [delimiter=/\r?\n/] - Delimiter to split the incoming buffer
+ */
+_Readable.prototype.split = function(delimiter) {
+	delimiter = delimiter || /\r?\n/;
+	return this.pipe(new SplitStream(delimiter, { readableObjectMode: true,  writableObjectMode: false }));
```

Comment:
The object mode specifications here shouldn't be necessary; SplitStream should set its own object mode properties because it only makes sense in one configuration.